tests/integration2/tests.mk

```diff
@@ -29,39 +29,32 @@ ifneq ($(KUBECONFIG),)
     INTEGRATION_TEST_KUBECONFIG = $(KUBECONFIG)
 endif
 
-# The names of the integration test folders at ROOT/tests/integration2/*.
-_INTEGRATION_TEST_NAMES = galley mixer citadel
-
-# Generate the names of the integration test targets that use local environment (i.e. test.integration.galley)
-_INTEGRATION_TESTS_LOCAL = $(addprefix test.integration., $(_INTEGRATION_TEST_NAMES))
-
-# Generate the names of the integration test targets that use kubernetes environment (i.e. test.integration.galley.kube)
-_INTEGRATION_TESTS_KUBE  = $(addsuffix .kube, $(addprefix test.integration., $(_INTEGRATION_TEST_NAMES)))
-
 # This is a useful debugging target for testing everything.
 .PHONY: test.integration.all
 test.integration.all: test.integration test.integration.kube
 
-# All integration tests targeting local environment.
-.PHONY: test.integration
-test.integration: $(_INTEGRATION_TESTS_LOCAL)
+JUNIT_UNIT_TEST_XML ?= $(ISTIO_OUT)/junit_unit-tests.xml
+JUNIT_REPORT = $(shell which go-junit-report 2> /dev/null || echo "${ISTIO_BIN}/go-junit-report")
 
-# All integration tests targeting Kubernetes environment.
-.PHONY: test.integration.kube
-test.integration.kube: $(_INTEGRATION_TESTS_KUBE)
+# TODO: Exclude examples and qualification since they are very flaky.
+TEST_PACKAGES = $(shell go list ./tests/integration2/... | grep -v /qualification | grep -v /examples)
 
-# Generate integration test targets for local environment.
```

Comment:
Please don't remove these targets. These create test.integration.component and test.integration.kube.component targets, which are very useful for running one-off integration tests.